src/index.html

```diff
@@ -12,7 +12,54 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__body">
+    <!-- <h1>Stars</h1> -->
```

Comment:
There's an empty line here. Remember to avoid empty lines between parent and child elements.